accessibility/README.md

```diff
@@ -9,3 +9,22 @@ AccessibilityServices services = AccessibilityServices.newInstance(context);
 services.isSpokenFeedbackEnabled();
 ```
 
+## Custom accessibility actions
+- a wrapper around View actions to facilitate actions via dialog or TalkBack local gestures menu
+
+First create `Actions`. It's necessary to give each `Action` a unique resource ID (required by `AccessibilityActionCompat` see `ActionsAccessibilityDelegate`), and a display label.
```

Comment:
01b5fc9

Only added a link to the android class, not the internal ones - people can use the github shortcut for that, else it's gonna be balls to maintain.